lib/utils.js

```diff
@@ -119,11 +119,23 @@ const makeFileDirectoryIfNeeded = function (filepath) {
   }
 };
 
+
+const getBorwser = async function(url, launchOptions) {
+  if(url){
+    var queryString = Object.keys(launchOptions).map(key => key + '=' + launchOptions[key]).join('&');
```

Comment:
Can you point to where query strings are described by the browser endpoint? You may want to use encodeURIComponent on the key or the value.

Reply:
It is specific to Browserless. I am not sure if the generic to puppeteer.
https://docs.browserless.io/docs/chrome-flags.html